src/transform/xsl/default.xsl

```diff
@@ -1588,12 +1588,10 @@
       </xsl:when>
       <xsl:otherwise>
         <xsl:for-each select="element-citation | nlm-citation | mixed-citation">
-            <p id="{@id}">
```

Comment:
I can see the `template match="mixed-citation">` duplicates this logic, but `element-citation` and `nlm-citation` do not. Those citations wouldn't render any HTML elements with an `id` so I suspect internal links would stop working those objects. Should we remove the redundant <p> tag from mixed-citation instead?